### ISSUE TITLE
smallrnaseq fastqc step failure

### ISSUE THREAD

**Issue by mshadbolt** [bcbio/bcbio-nextgen#2525]
Hi
I'm trying to run the small rna seq pipeline on a set of fastq files

The final error I get is:
```
[2018-09-25T10:26Z] Skipping '~/work/align/SRR5234257_trimmed/SRR5234257_trimmed_star/SRR5234257_trimmed.bam' which didn't exist, or couldn't be read
Traceback (most recent call last):
  File "~/bcbio-nextgen/tools/bin/bcbio_nextgen.py", line 242, in <module>
    main(**kwargs)
  File "~/bcbio-nextgen/tools/bin/bcbio_nextgen.py", line 47, in main
    run_main(**kwargs)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 45, in run_main
    fc_dir, run_info_yaml)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 89, in _run_toplevel
    for xs in pipeline(config, run_info_yaml, parallel, dirs, samples):
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 351, in smallrnaseqpipeline
    samples = qcsummary.generate_parallel(samples, run_parallel)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/pipeline/qcsummary.py", line 39, in generate_parallel
    qced = run_parallel("pipeline_summary", to_analyze)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/distributed/multi.py", line 28, in run_parallel
    return run_multicore(fn, items, config, parallel=parallel)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/distributed/multi.py", line 86, in run_multicore
    for data in joblib.Parallel(parallel["num_jobs"], batch_size=1)(joblib.delayed(fn)(x) for x in items):
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 779, in __call__
    while self.dispatch_one_batch(iterator):
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 625, in dispatch_one_batch
    self._dispatch(tasks)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 588, in _dispatch
    job = self._backend.apply_async(batch, callback=cb)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/joblib/_parallel_backends.py", line 111, in apply_async
    result = ImmediateResult(func)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/joblib/_parallel_backends.py", line 332, in __init__
    self.results = batch()
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 131, in __call__
    return [func(*args, **kwargs) for func, args, kwargs in self.items]
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/utils.py", line 52, in wrapper
    return apply(f, *args, **kwargs)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/distributed/multitasks.py", line 208, in pipeline_summary
    return qcsummary.pipeline_summary(*args)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/pipeline/qcsummary.py", line 70, in pipeline_summary
    data["summary"] = _run_qc_tools(work_bam, work_data)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/pipeline/qcsummary.py", line 161, in _run_qc_tools
    out = qc_fn(bam_file, data, cur_qc_dir)
  File "~/bcbio-nextgen/data/anaconda/lib/python2.7/site-packages/bcbio/qc/fastqc.py", line 64, in run
    raise ValueError("FastQC failed to produce output HTML file: %s" % os.listdir(tx_tmp_dir))
ValueError: FastQC failed to produce output HTML file: []
```

I checked the align folder and strangely that file is indeed missing, but there were the files `seqs.bam` and `seqs.bam.bai` in the align folder. I had a look at these files and they are the missing ones so maybe there is a bug in the code that needs to be fixed. I manually moved the files to the expected directory and am re-running to see if it will proceed past this point. This is the first sample alphabetically if that helps.

**Comment by mshadbolt** [comment]
Nope my manual sneaky fix didn't work, it re-ran the STAR alignment for that file, deleted the folder I made and again I have `seqs.bam` and `seqs.bam.bai` in the align folder so the pipeline can't find the real files. If anyone could help me out that would be great.

Cheers

**Comment by lpantano** [comment]
Hi,

I pushed a fix that may fix your issues as well (7170c5a25e444030f03708ff2223d4ea482a0d71). But you would need to update to latest development version.

Let me know.

**Comment by roryk** [comment]
Thanks, looks like Lorena fixed this so should be all set.
